### PR TITLE
[Bug fix] Populate parameter_type_category_rel table with imaging and electrophysiology variables

### PR DESCRIPTION
**Pull request by cmadjar**
### Brief summary of changes

The SQL patch attached to this PR will fix the issue that imaging or electrophysiology `parameter_type` fields are not linked to a parameter type category via the `parameter_type_category_rel` table. 

This PR goes with the LORIS-MRI PR: https://github.com/aces/Loris-MRI/pull/370

### This resolves issue...

- [x] Issue reported in the following PR: https://github.com/aces/Loris/pull/3695#discussion_r242642123

### To test this change...

- [ ] run the SQL patch on your sandbox with imaging data

### Caveat for existing projects

Make sure to source the SQL patch to fix existing projects' table.

### CODE REVIEW

**Comment by cmadjar** [comment]
@driusan in the end, I sent the PR to minor as if I remember correctly, there should not be any SQL in bug fix. I can always rebase it if needed. Just let me know.

@ridz1208 @driusan I put priority high label as @davidblader PR might need this bug fix for his PR https://github.com/aces/Loris/pull/3695

**Comment by driusan** [comment]
Priority: High is for roadmap planning, not interdependencies.

File: SQL/New_patches/2018-01-16_map_parameter_type_and_parameter_type_category_for_imaging.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+INSERT INTO parameter_type_category_rel (ParameterTypeID, ParameterTypeCategoryID)
+  SELECT ParameterTypeID, ParameterTypeCategoryID
```
Comment (driusan) [inline SQL/New_patches/2018-01-16_map_parameter_type_and_parameter_type_category_for_imaging.sql]:
```suggestion
  SELECT DISTINCT ParameterTypeID, ParameterTypeCategoryID
```

Comment (cmadjar) [reply SQL/New_patches/2018-01-16_map_parameter_type_and_parameter_type_category_for_imaging.sql]: @driusan Thank you for the suggestion. Implemented in the next commit. I also renamed the patch with the proper year ;).

**Pull request merged**